drivers/docker/driver.go

```diff
@@ -740,6 +740,20 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		Config: driverConfig.Logging.Config,
 	}
 
+	if hostConfig.LogConfig.Type == "json-file" || hostConfig.LogConfig.Type == "" {
+		logger.Debug("configuring docker json-file logs")
+		hostConfig.LogConfig.Type = "json-file"
+		if hostConfig.LogConfig.Config == nil {
+			hostConfig.LogConfig.Config = make(map[string]string)
+		}
+		hostConfig.LogConfig.Config["max-file"] = "3"
+		hostConfig.LogConfig.Config["max-size"] = "10m"
```

Comment:
Minimize these resources by default as they're in addition to the logs Nomad and only used as a buffer during agent restarts. (If the operator wants to use them directly they can manually configure them.)

Maybe:
```go
		hostConfig.LogConfig.Config["max-file"] = "2"
		hostConfig.LogConfig.Config["max-size"] = "2m"
```

Reply:
fixed